src/core_plugins/metrics/public/directives/visualization.js

```diff
@@ -19,22 +19,24 @@ app.directive('metricsVisualization', (timefilter) => {
 
       // For Metrics, Gauges and markdown visualizations we want to hide the
       // panel title because it just doens't make sense to show it.
-      const panel = $($el[0]).parents('.panel');
-      if (panel.length) {
-        const panelHeading = panel.find('.panel-heading');
-        const panelTitle = panel.find('.panel-title');
-        const matchingTypes = ['metric', 'gauge', 'markdown'];
-        if (panelHeading.length && panelTitle.length && _.contains(matchingTypes, $scope.model.type)) {
-          panel.css({ position: 'relative' });
-          panelHeading.css({
-            position: 'absolute',
-            top: 0,
-            right: 0,
-            zIndex: 100
-          });
-          panelTitle.css({ display: 'none' });
+      $timeout(() => {
```

Comment:
Can you add a little comment motivating why we unhook this block from the stack? It makes sense in the context of the PR, but may not after it is merged.